animations/analysis_options.yaml

```diff
@@ -7,25 +7,19 @@ analyzer:
 
 linter:
   rules:
-    - avoid_types_on_closure_parameters
-    - avoid_void_async
-    - await_only_futures
-    - camel_case_types
-    - cancel_subscriptions
-    - close_sinks
-    - constant_identifier_names
-    - control_flow_in_finally
-    - directives_ordering
-    - empty_statements
-    - hash_and_equals
-    - implementation_imports
-    - non_constant_identifier_names
-    - package_api_docs
-    - package_names
-    - package_prefixed_library_names
-    - test_types_in_equals
-    - throw_in_finally
-    - unnecessary_brace_in_string_interps
-    - unnecessary_getters_setters
-    - unnecessary_new
-    - unnecessary_statements
+    avoid_types_on_closure_parameters: true
+    avoid_void_async: true
+    await_only_futures: true
+    camel_case_types: true
```

Comment:
Several of these seem to already be in the lint sets, so can't remove them from here?

Reply:
Which lint set did I miss? I've trimmed some from our previous list of lints

Reply:
I just spot checked. But for example `camel_case_types` is in https://github.com/dart-lang/lints/blob/main/lib/core.yaml

Reply:
By lints sets, I meant is already in one of these three files (which are chained together by imports):
https://github.com/dart-lang/lints/blob/main/lib/core.yaml
https://github.com/dart-lang/lints/blob/main/lib/recommended.yaml
https://github.com/flutter/packages/blob/master/packages/flutter_lints/lib/flutter.yaml

Reply:
Looks like `hash_and_equals` and `non_constant_identifier_names` are also now duplicates.